Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "serde_urlencoded"
-version = "0.5.5"
+version = "0.5.6"
```

Comment:
There are various `url` items that appear in the public API, so AFAIK this is a breaking change.

Reply:
Oh indeed, I misread the `rg` output.